templater/jobs/presubmit/eks-anywhere-build-tooling/eks-a-upgrader-image-presubmits.yaml

```diff
@@ -0,0 +1,13 @@
+jobName: eks-a-upgrader-image-presubmits
+runIfChanged: EKS_DISTRO_MINIMAL_BASE_TAG_FILE|EKSD_LATEST_RELEASES|^build/lib/.*|projects/aws/upgrader/.*|projects/containerd/.*|projects/kubernetes-sigs/cri-tools/.*
+commands:
+- if $(make check-project-path-exists); then make build -C $PROJECT_PATH; fi
+projectPath: projects/aws/upgrader
+imageBuild: true
+envVars:
+- name: ARTIFACTS_BUCKET
+  value: s3://projectbuildpipeline-857-pipelineoutputartifactsb-10ajmk30khe3f
+resources:
+  requests:
+    memory: 16Gi
```

Comment:
lets lower these, look at some other jobs for a smaller configuration, like a 2/8 or something.